### DISCUSSION TITLE
beta  badge

### DISCUSSION THREAD

**Discussion by Muratwin3439** [community/community#38346] [Other Feature Feedback, Questions, & Ideas]
![image](https://user-images.githubusercontent.com/74829131/200113640-fd13dc18-18c1-42cf-a1fc-1dbb99e3816d.png)
How can I get this badge?
How old are friends who know this badge?

**Comment by airtower-luna** [comment]
That's not a badge, it marks the "Achievements" section of the profile as being in beta. It should show if you have any achievement, as long as you don't disable showing your achievements.

**Reply by Muratwin3439** [reply]
![image](https://user-images.githubusercontent.com/74829131/200114262-d5cc5c73-ef45-4721-a68f-644fb100fa81.png)
So how is this taken?

**Reply by airtower-luna** [reply]
By paying for a "pro" account: https://docs.github.com/en/get-started/learning-about-github/githubs-products#github-pro

I believe there's a option for students to get it for free for a while, but it's been a few years since I graduated, so I haven't looked deeper into it. :grin:

**Reply by Muratwin3439** [reply]
how do i apply

**Reply by DSDmark** [reply]
Go there and fill all details they ask about you https://education.github.com/pack
